silk/tests/silk_run.py

```diff
@@ -185,6 +189,9 @@ def parseArgs(self, argv):
             help='Set the verbosity level of the console (0=quiet, 1=default, 2=verbose)')
         parser.add_argument('pattern', nargs='+', metavar='P',
             help='test file search pattern')
+        parser.add_argument('--otns', dest='otns_server',
```

Comment:
add  '-o' to be consistent with other arguments.

Reply:
Originally added, but removed since Simon commented and said `-o` is too comment and might lead to confusion.

Reply:
`-o` might look like an output argument.

Reply:
I don't have a strong preference either way, so I can add it if you think it's better.

Reply:
let's use '-s'